deploy/README.md

```diff
@@ -69,6 +69,7 @@ To deploy this project to  server:
 3. Obtain a copy of the `ansible-vault-password.txt` file in LastPass and place it in the directory which contains this document.
 4. Install [Ansible](https://www.ansible.com/) version 2.11. Instructions for macOS are as follows:
   - Install Ansible at the specific 2.11 version: `python3 -m pip install --user ansible-core==2.11.1`
+  - Install `ansible.posix` to make use of the [ansible.posix.synchronize](https://docs.ansible.com/ansible/latest/collections/ansible/posix/synchronize_module.html#ansible-posix-synchronize-module-a-wrapper-around-rsync-to-make-common-tasks-in-your-playbooks-quick-and-easy) module: `ansible-galaxy collection install ansible.posix`
```

Comment:
Thanks for catching this! We should put this step *after* sourcing the shell's RC file (otherwise, the invocation of `ansible-galaxy` won't resolve)

Reply:
Good catch on my catch! Done in https://github.com/w3c/aria-at-app/pull/912/commits/b51576544cc9b5f8e9f4f26c705a5bd1030f0609